packager/media/base/muxer_util.cc

```diff
@@ -86,11 +90,11 @@ bool ValidateSegmentTemplate(const std::string& segment_template) {
     }
   }
   if (has_number && has_time) {
-    LOG(ERROR) << "SegmentTemplate: $Number$ and $Time$ should not co-exist.";
+    LOG(ERROR) << "SegmentTemplate: $Number$ and $Time$/$DecodeTime$ should not co-exist.";
     return false;
   }
   if (!has_number && !has_time) {
-    LOG(ERROR) << "SegmentTemplate: $Number$ or $Time$ should exist.";
+    LOG(ERROR) << "SegmentTemplate: $Number$ or $Time$/$DecodeTime$ should exist.";
```

Comment:
We need to figure out exactly how we want to validate these, because we're deviating from ISO/IEC 23009-1.